docs/database/database-metrics.md

```diff
@@ -440,6 +445,313 @@ This metric is [recommended][MetricRecommended].
 | [`db.client.connection.pool.name`](/docs/attributes-registry/db.md) | string | The name of the connection pool; unique within the instrumented application. In case the connection pool implementation doesn't provide a name, instrumentation SHOULD use a combination of parameters that would make the name unique, for example, combining attributes `server.address`, `server.port`, and `db.namespace`, formatted as `server.address:server.port/db.namespace`. Instrumentations that generate connection pool name following different patterns SHOULD document it. | `myDataSource` | `Required` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
 
+<!-- markdownlint-restore -->
+<!-- prettier-ignore-end -->
+<!-- END AUTOGENERATED TEXT -->
+<!-- endsemconv -->
+
+## Azure Cosmos DB
+
+The following metric instruments describe Azure Cosmos DB client behaviour.
+
+### Metric: `db.client.operation.duration`
+
+It captures the total time taken by an Azure Cosmos DB operation. This metric follows the common [db.client.operation.duration] (#metric-dbclientoperationduration)b definition.
```

Comment:
each metric should have a requirement level - take a look at the examples of existing metrics in this file. I assume all of them should be required or recommended (when available) for cosmos?